metrics/http_middleware.lua

```diff
@@ -71,6 +71,9 @@ end
 -- ... arguments for pcall to instrument
 function export.observe(collector, route, ...)
     return collector:observe_latency(function(ok, result)
+        if result == nil then
```

Comment:
non-table result will fail with `trying to subscribe a string value`, so I think `type(result) == 'table'` is the condition.